src/keen-slider.js

```diff
@@ -159,39 +159,32 @@ function KeenSlider(initialContainer, initialOptions = {}) {
   }
 
   function eventsAdd() {
-    const isEventPassive = !(!isTouchable() || !touchActive)
-
     eventAdd(window, 'orientationchange', sliderResizeFix)
     eventAdd(window, 'resize', () => sliderResize())
-    eventAdd(container, 'dragstart', function (e) {
-      if (!isTouchable()) return
-      e.preventDefault()
-    }, {passive: isEventPassive})
     eventAdd(container, 'mousemove', eventDrag)
+    eventAdd(container, 'mouseleave', eventDragStop)
+    eventAdd(container, 'mouseup', eventDragStop)
     eventAdd(container, 'mousedown', eventDragStart, {
       passive: true,
     })
     eventAdd(container, 'touchstart', eventDragStart, {
       passive: true,
     })
-    eventAdd(container, 'mouseleave', eventDragStop, {
-      passive: true,
-    })
-    eventAdd(container, 'mouseup', eventDragStop, {
-      passive: true,
-    })
     eventAdd(container, 'touchend', eventDragStop, {
       passive: true,
     })
     eventAdd(container, 'touchcancel', eventDragStop, {
       passive: true,
     })
     eventAdd(container, 'touchmove', eventDrag, {
-      passive: isEventPassive,
-    })
-    eventAdd(window, 'wheel', eventWheel, {
-      passive: isEventPassive,
+      passive: false,
```

Comment:
Do we have to explicitly identify non-passive event listeners? I believe this is default behaviour

Reply:
i simply followed what was in the `5.2.2`. I believe that the creator of the library was planning to implement a sliding feature based on the wheel event

Reply:
If this is an upstream change, it might be worth merging the forked repo into this before making other changes, otherwise it's hard to see what changes you're making from the source repo.